ingest-app/test/cmr/ingest/test/services/jobs.clj

```diff
@@ -3,16 +3,16 @@
   (:require
     [clojure.test :refer :all]
     [cmr.common.util :as u :refer [are3]]
-    [cmr.ingest.services.jobs :as jobs]))
+    [cmr.ingest.services.email-processing :as email-processing]))
 
 (deftest create-query-params
```

Comment:
Move email processing related tests to `cmr.ingest.test.services.email-processing` namespace.